### DISCUSSION TITLE
wikiimage_character produces inaccurate images

### DISCUSSION THREAD

**Discussion by HawkinsThe71st** [FakeFishGames/Barotrauma#13642] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Using the wikiimage_character command on most characters results in incorrectly scaled, cropped or blank images depending on the creature.
Some examples:

Latcher
![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/ed0c7c03-b016-4b4d-80b4-2c0ef9efa7e2)

Husk
![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/66f0f8e8-aa43-447c-b1c7-a5da0d0e5925)

Hammerhead
![image](https://github.com/FakeFishGames/Barotrauma/assets/115734437/37396e84-8508-4ee6-9ba7-dc857649d60e)

### Reproduction steps

1. control a character
2. use wikiimage_character
3. examine result

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.2.8.0 (Winter Update hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ToumanLin** [comment]
For me I suggest it is a serious problem as I need to use this order to generate pics for my new characters...

**Comment by Regalis11** [comment]
(Finally) fixed in our private repo, fix coming up in the next unstable build and the upcoming release.
